addons/l10n_dk/data/account_tag_data.xml

```diff
@@ -0,0 +1,131 @@
+        <field name="name">Ekstraordinære udgifter</field>
+        <field name="applicability">accounts</field>
+    </record>
+
+    <!--Aktiver-->
+    <record id="tag_goodwill_licenses_etc" model="account.account.tag">
+        <field name="name">Goodwill, licenser mm.</field>
+        <field name="applicability">accounts</field>
+    </record>
+    <record id="tag_land_buildings" model="account.account.tag">
+        <field name="name">Grunde og bygninger</field>
+        <field name="applicability">accounts</field>
+    </record>
+    <record id="tag_other_equipment_tools_inventory" model="account.account.tag">
+        <field name="name">Andre anlæg, driftsmateriel og inventar</field>
+        <field name="applicability">accounts</field>
+    </record>
+    <record id="tag_inventories" model="account.account.tag">
+        <field name="name">Varebeholdninger</field>
+        <field name="applicability">accounts</field>
+    </record>
+    <record id="tag_work_in_progress" model="account.account.tag">
+        <field name="name">Igangværende arbejder</field>
+        <field name="applicability">accounts</field>
+    </record>
+    <record id="tag_other_receivables" model="account.account.tag">
+        <field name="name">Andre tilgodehavender</field>
+        <field name="applicability">accounts</field>
+    </record>
+
+    <!--Passiver-->
+    <record id="tag_equity_individual_enterprices" model="account.account.tag">
+        <field name="name">Egenkapital enkeltmandsvirksomheder</field>
+        <field name="applicability">accounts</field>
+    </record>
+    <record id="tag_equity_corporations" model="account.account.tag">
+        <field name="name">Egenkapital selskaber</field>
+        <field name="applicability">accounts</field>
+    </record>
+    <record id="tag_credit_institutions" model="account.account.tag">
+        <field name="name">Kreditinstitutter</field>
+        <field name="applicability">accounts</field>
+    </record>
+    <record id="tag_bank_loans" model="account.account.tag">
+        <field name="name">Bankgæld</field>
+        <field name="applicability">accounts</field>
+    </record>
+    <record id="tag_prepayments_from_customers" model="account.account.tag">
+        <field name="name">Forudbetalinger fra kunder</field>
+        <field name="applicability">accounts</field>
+    </record>
+    <record id="tag_vat_taxes" model="account.account.tag">
+        <field name="name">Moms og afgifter</field>
+        <field name="applicability">accounts</field>
+    </record>
+    <record id="tag_payroll_related_liabilities" model="account.account.tag">
+        <field name="name">Løn relateret gæld</field>
+        <field name="applicability">accounts</field>
+    </record>
+    <record id="tag_other_current_liabilities" model="account.account.tag">
+        <field name="name">Anden kortfristet gæld</field>
+        <field name="applicability">accounts</field>
+    </record>
+
+    <!-- Account tag -->
```

Comment:
How are the account tags in this section used ? The names are in German and not related to the Danish COA or reporting as far as i can see...

Reply:
these are nothing but adaptation to account tags of what previously were `account.account.type` records <a href="https://github.com/odoo/odoo/pull/23272/files#diff-4bb687b273f7f89768330e567808d225R12">here</a> in your previous PR(V11). Looks like developer mistakenly took German translation(de.po) instead of Danish(da.po) for it's `name`. They are in-fact used to generate financial reports in enterprise now, you can see it here https://github.com/odoo/enterprise/compare/12.0...odoo-dev:master-l10n_dk-dat if you have access it.
The output of those reports matches with the ones extracted from your V11 branch. So functionally speaking, nothing much has changed from your V11 branch, but technically, few things does.